pythonsdk/requirements.txt

```diff
@@ -1,15 +1,154 @@
-# Make sure we have a reasonably current version of pip installed
-pip>=23.1.2
-
-# These are the primary requirements
-fastapi==0.99.1
-uvicorn[standard]==0.22.0
-
-# For testing
+#
```

Comment:
Best practice in python land seems to be to check this in, even though it is generated via `pip-compile`.  So that's what we do ;).